ckanext/canada/templates/admin/snippets/publish_package_list.html

```diff
@@ -0,0 +1,34 @@
+{% set list_class = 'list-bullet-none' %}
+
+{% if packages %}
+  <form action="{{ h.url_for('canada.ckanadmin_publish_datasets') }}" method="POST" id="publish_form">
+
+    <div class="mrgn-bttm-md" style="padding-top: 25px;">
+      <label for="publish_all" class="form-checkbox no-colon" style="float:right; display:inline-block; margin-top:-25px;">
+        <input type="checkbox" id="publish_all" name="publish_all"> {{ _('Select All for Publishing') }}
+      </label>
+    </div>
+
+    <ul class="list-unstyled">
+      {% for package in packages %}
+        {% snippet 'admin/snippets/package_item.html', package=package, item_class=item_class, hide_resources=hide_resources, banner=banner, truncate=truncate, truncate_title=truncate_title %}
+      {% endfor %}
+    </ul>
+
+    <fieldset class="admin-publish-date-fieldset mrgn-tp-md">
+      <div>
+        <label for="publish_date" class="admin-publish-date-label">
+          {% if h.lang() == 'en' %}
+            Publish Date (<abbr title="Four digits year">YYYY</abbr>-<abbr title="Two digits month">MM</abbr>-<abbr title="Two digits day">DD</abbr>):
+          {% else %}
+            Date de publication <abbr title="Quatre chiffres de l'année">(AAAA</abbr>-<abbr title="Deux chiffres du mois">MM</abbr>-<abbr title="Deux chiffres du jour">JJ</abbr>) :
+          {% endif %}
```

Comment:
`{% trans %}` is good for marking translated blocks that have tags in them

Reply:
This is now in PO file. and I also looked for any other places with these silly IF statements. Didn't find anymore, but was able to clean up some of the `if lang == 'en'` statements a bit.